src/etc/inc/certs.inc

```diff
@@ -330,7 +330,7 @@ function cert_get_subject($str_crt, $decode = true)
         }
     }
 
-    return $subject;
+    return $subject ?? '';
```

Comment:
is that a clever "if not defined syntax"? I would prefer a $subject = ''; definition at the top

Reply:
https://secure.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.coalesce